jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/util/CodeWhispererFileContextProvider.kt

```diff
@@ -219,7 +219,7 @@ class DefaultCodeWhispererFileContextProvider(private val project: Project) : Fi
             contentToChunk[bm25Result.docString]?.let {
                 Chunk(content = it.nextChunk, path = it.path, score = bm25Result.score)
             }
-        }
+        }.filter { it.content.isNotBlank() }
```

Comment:
you should probably not even construct the chunk if the content is blank
saves some memory overhead